packages/volto/src/components/manage/UniversalLink/UniversalLink.tsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import { HashLink as Link } from 'react-router-hash-link';
+import { useSelector } from 'react-redux';
+import {
+  flattenToAppURL,
+  isInternalURL,
+  URLUtils,
+} from '@plone/volto/helpers/Url/Url';
+import config from '@plone/volto/registry';
+import cx from 'classnames';
+
+interface UniversalLinkProps {
+  href: string;
```

Comment:
href is optional too, it's either href or item.

Reply:
Ok i use now Union Types here to differentiate between href or item.